app/Http/Requests/Match/CreateMatchRequest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Http\Requests\Match;
+
+use Illuminate\Foundation\Http\FormRequest;
+use App\Models\Match;
+
+class CreateMatchRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
You need to add user authorization here - make sure that the user is logged in. https://laravel.com/docs/5.4/validation#authorizing-form-requests